site.yml

```diff
@@ -0,0 +1,51 @@
+site:
+  title: "Open Manufacturing Platform - Semantic Data Structuring Working Group"
+  url: https://openmanufacturingplatform.github.io/sds-documentation/
+  start_page: sds-documentation::index.adoc
+content:
+  sources:
+     - url: .
+       branches: HEAD
+       tags: '*'
```

Comment:
I wonder if this line should be removed. The sds-documentation itself is not going to be tagged, is it?